apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts

```
@@ -330,6 +395,21 @@ export class DsbApiService implements OnModuleInit {
     };
   }
 
+
+  public async getApplicationsByOwner(ownerDid: string): Promise<Array<IApp>> {
+    await this.enableTLS();
+    try {
+      const data = await this.iamService.getApplicationsByOwner(ownerDid)
+      return data
+    } catch (e) {
+      if (e.response) {
+        this.logger.error(e);
```

why not rethrow this error? either return null in that case and update method definition or rethrow this error